src/Microsoft.ML.Core/Microsoft.ML.Core.csproj

```diff
@@ -10,7 +10,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.ML.DataView\Microsoft.ML.DataView.csproj" />
-    
+
```

Comment:
I think you can revert all these .csproj files. No need cluttering the history making unnecessary changes.